src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
Don't we *want* to be using the relative path to distinguish from an absolute path? It seems like we *wouldn't* want a path starting with a `/` to resolve relative to the current folder.